examples/custom_sliders/custom_sliders.c

```diff
@@ -75,7 +75,7 @@ int main()
             // raygui: controls drawing
             //----------------------------------------------------------------------------------
             GuiGroupBox((Rectangle){ 66, 24, 276, 312 }, "STANDARD");
-            value = GuiSlider((Rectangle){ 96, 48, 216, 16 }, TextFormat("%0.2f", value), NULL, value, 0.0f, 1.0f);
+            value = GuiSlider((Rectangle){ 96, 48, 216, 16 }, TextFormat("%0.2f", value), NULL, &value, 0.0f, 1.0f);
```

Comment:
Ouch! raylib 4.0 redesign! Actually return value is not `value` anymore, it's intended for state info.